public/manifest.json

```diff
@@ -1,25 +1,25 @@
 {
```

Comment:
Manifest is also not really needed for us, but if you want to keep and maintain that, it's up to you